engine/src/python/unit_wrapper_class.h

```diff
@@ -162,15 +167,15 @@ class UnitWrapper : public UnitContainer {
     boost::python::tuple GetOrientation() {
         {
         CHECKME VS_BOOST_MAKE_TUPLE(VS_BOOST_MAKE_TUPLE(0, 0, 0), VS_BOOST_MAKE_TUPLE(0, 0, 0), VS_BOOST_MAKE_TUPLE(0,
-                    0,
```

Comment:
Probably better to just not break the line or break earlier for easier reading. Same for the rest of the file.